test/tests/reporter/GriddedDataReporter/gold/griddedData_out.json

```diff
@@ -0,0 +1,94 @@
+{
+    "reporters": {
+        "parameter_info": {
+            "type": "GriddedDataReporter",
+            "values": {
+                "axes": {
+                    "type": "std::vector<int>"
+                },
+                "dim": {
+                    "type": "unsigned int"
+                },
+                "gird": {
```

Comment:
grid

Reply:
> Will you add an example that has x,y, and time so that we get an axes=[0,1,3]. Use one from the PiecewiseMultiInterpolation function tests.

I modified the test so that it has x,y,t.

Reply:
> grid

done